### DISCUSSION TITLE
Incomplete Address Paradigm?

### DISCUSSION THREAD

**Discussion by swartik** [CommonCoreOntology/CommonCoreOntologies#393] [Announcements]
I was using the paradigm for a person's address in _Modeling Information with the Common Core Ontologies_ (Section 4.1). It seems to me that the pattern is incomplete. It locates the geospatial region for a zip code within a state. The thing is, many such geospatial regions are located within a state – within a city, for that matter. What is the relationship between a residential facility and its zip code? Is the picture missing an additional located_in between those two individuals?

**Comment by rorudn** [comment]
The facility is located at some lot of land that is identified by the facility address. I would say I'm sure of that but there may be farms that are identified only rural route. That exception leads me to think that the person/organization/facility name is also a part of the address. Past that, the spatial relationships between lot, city, and state are pretty well behaved although there are counter-examples to be found. ZIP codes are not so well-behaved, some span multiple states and some identify organizations more that a land area (Smokey the Bear has one, 20252). So I do think its safe to say that a facility location is a proper spatial part of a region identified by ZIP code but adding those assertions don't seem to me to strengthen the identifying relation between the full address and the facility.

**Comment by swartik** [comment]
I agree, but I guess I didn't make my point well. I want to write a SPARQL query that retrieves a residence's street, city, state, and zip code.  If a state (or in my case, a city) is the object of multiple located-in relationships with a geospatial region as the subject, the query will return a result for each geospatial region. Asserting which geospatial region the residential facility is located in would resolve the ambiguity. Is there another way?

Also, I just noticed that the last sentence in the first paragraph of Section 4 has some extra words:

> The examples _were selected are intended to_ illustrate common mapping problems.

You could drop “are intended” or “were selected”.

**Comment by neilotte** [comment]
@swartik I believe the issues raised here warrant a more comprehensive review and discussion. Hence moving to discussion forum now.
